src/phone-input/__tests__/phone-input.e2e.js

```diff
@@ -0,0 +1,96 @@
+/*
+Copyright (c) 2018-2019 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+/* eslint-env node */
+/* eslint-disable flowtype/require-valid-file-annotation */
+/* global document */
+
+const {mount, analyzeAccessibility} = require('../../../e2e/helpers');
+
+const selectors = {
+  phoneInput: `[data-baseweb="phone-input"]`,
+  phoneInputInput: `[data-baseweb="phone-input"] > input`,
+  phoneInputFlag: `[data-e2e="country-flag"]`,
+  phoneInputDialcode: `[data-e2e="phone-input-dialcode"]`,
+  phoneInputSelect: `[data-baseweb="select"]`,
+  phoneInputSelectDropdown: `[role="listbox"]`,
+  phoneInputSelectListItem: `[data-e2e="country-select-list-item"]`,
+};
+
+const countryListItemForIso = iso =>
+  `${selectors.phoneInputSelectListItem}[data-iso="${iso}"]`;
+
+const unitedStates = {iso: 'US', dialCode: '+1'};
+const unitedKingdom = {iso: 'GB', dialCode: '+44'};
+
+describe('PhoneInput', () => {
+  beforeEach(async () => {
+    await mount(page, 'phone-input');
+    await page.waitFor(selectors.phoneInput);
+  });
+
+  it('passes basic a11y tests', async () => {
+    const accessibilityReport = await analyzeAccessibility(page);
```

Comment:
Need to wait for the component rendered, not page, to make sure accessibility check runs on the component.

Reply:
oh i moved that into the `beforeAll`